package.json

```diff
@@ -19,11 +19,13 @@
     "test": "npm run test:unit"
   },
   "dependencies": {
+    "chart.js": "^3.8.0",
     "formik": "^2.2.9",
     "framer-motion": "^6.4.3",
     "mongoose": "^6.4.0",
     "react": "18.1.0",
     "react-beautiful-dnd": "^13.1.0",
+    "react-chartjs-2": "^4.3.1",
```

Comment:
I'm not use about this, but do you need  and used booth, `chart.js `and `react-chartjs-2` 🤔